documents/entity/service.go

```diff
@@ -210,14 +213,64 @@ func (s service) DeriveEntityResponse(model documents.Model) (*cliententitypb.En
 		return nil, err
 	}
 
+	entityID := model.ID()
+	var relationships []*cliententitypb.Relationship
+	// if this identity is not the owner of the entity, return an empty relationships array
+	selfDID, err := contextutil.AccountDID(ctx)
+	if err != nil {
+		return nil, errors.New("failed to get self ID")
+	}
+
+	isCollaborator, err := model.IsDIDCollaborator(selfDID)
+	if err != nil {
+		return nil, err
+	}
+	if !isCollaborator {
+		return &cliententitypb.EntityResponse{
+			Header: h,
+			Data: &cliententitypb.EntityDataResponse{
+				Entity:        data,
+				Relationships: relationships,
+			},
+		}, nil
+	}
+	_, models, err := s.ListEntityRelationships(ctx, entityID)
+	if err != nil {
+		return nil, err
+	}
+
+	//list the relationships associated with the entity
+	if models != nil {
+		for _, m := range models {
+			active := false
+			tokens, err := m.GetAccessTokens()
+			if err != nil {
+				return nil, err
+			}
+			if len(tokens) != 0 {
+				active = true
+			}
+			targetDID := m.(*entityrelationship.EntityRelationship).TargetIdentity.String()
+			r := &cliententitypb.Relationship{
+				Identity: targetDID,
+				Active:   active,
```

Comment:
This can be simplified to: 
```suggestion
				Active:   (len(tokens) != 0),
```

Reply:
👍